### DISCUSSION TITLE
Setting Up Brotli Compression in NextJS with Custom Server

### DISCUSSION THREAD

**Discussion by amit-sharechat** [vercel/next.js#16627] [Help]
I am using **brotli-webpack-plugin**  to generate Brotli Files & **express-static-gzip** to serve the files.

Webpack is generating the files but server is not serving the Brotli files instead returns the Gzip files.

In next.config.js

```
conf.plugins.push(
			new BrotliPlugin({
				filename: "[path].br[query]",
				test: /\.js$|\.css$|\.html$/,
				threshold: 10240,
				minRatio: 0.8,
			}),
		);
```

In server.js

```
.use(
			"/",
			expressStaticGzip(BUILD_DIR, {
				enableBrotli: true,
				index: false,
				orderPreference: ["br"],
			}),
		)
```

Can anyone help me in this ?

**Comment by amirjr1378** [comment]
yeah iam wondering too.

**Reply by VBagmut** [reply]
Next.js is using built-in gzip compression. I think you need to disable it and then use your custom solution.
https://nextjs.org/docs/api-reference/next.config.js/compression

**Reply by dipmfsi** [reply]
@amit-sharechat Have you find the solution? If yes, Then please let us know

**Reply by hmellahi** [reply]
@amirjr1378  @dipmfsi any news?

**Comment by JollyGrin** [comment]
Also trying to load a unity webgl experience and having issues getting a working config file to let the `.br` files be loaded properly (from my public folder)

Anyone have any success?

**Comment by QuentinScDS** [comment]
pnpm approve-builds
There are no packages awaiting approval

And there is a lot of warning/errors during the data loading in Directus

**Comment by upewasuf20-design** [comment]
Informiere dich vor dem Spielen über verfügbare Angebote, Teilnahmebedingungen und geltende Vorschriften. Nur für Erwachsene ab dem gesetzlichen Mindestalter. Glücksspiel kann süchtig machen – spiele verantwortungsvoll, setze dir [gamblezen](https://gamblezencasino-de.de/) persönliche Grenzen und behalte deine Ausgaben im Blick.
